installer/default_tenant.sh

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#!/bin/bash
+
+DOMAIN_NAME=$1
+DEFAULT_TENANT_NAME=$2
```

Comment:
since we're exporting DEFAULT_TENANT_NAME we could just use:
`DEFAULT_TENANT_NAME=$DEFAULT_TENANT_NAME`